### PR TITLE
FT-38 FT Swagger: Advpoll changes

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/FT-38

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -2974,6 +2974,40 @@ paths:
...
+        schema:
+          type: string
+          format: uuid
+    post:
+      tags:
+        - 'group polls'
+      description: 'Vote in a group poll.'
+      responses:
+        '201':
+          description: 'The vote in a group poll was successfull.'
```
Comment (ParryLuke) [inline api.swagger.yaml:2992]: typo - successful

Comment (ParryLuke) [reply api.swagger.yaml:2992]: in a -> in the?

**Review by ParryLuke — CHANGES_REQUESTED** [review]
Will the changes to the other endpoints be added in this PR as well?

File: api.swagger.yaml (hunk 1 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: Fixed types, yes the changes to feeds are being done in FT-16 (I put a note in the task desc)

**Review by ParryLuke — APPROVED** [review]

**Comment by MireilleHelle** [comment]
Feeds project Swagger changes were merged under https://github.com/invotra/api/pull/343. PR no longer required. Closing.

**Pull request closed without merging**